src/Zinc-Tests/ZnEasyTest.class.st

```diff
@@ -44,6 +44,9 @@ ZnEasyTest >> testGetJpeg [
 { #category : #testing }
 ZnEasyTest >> testGetPng [
 	| url result |
+	self flag: #flakyTest.
+	self skip.
```

Comment:
Idea is to skip flaky tests and put a flag for that. So it can be easily found in the image.